bin/jobs_functions.js

```diff
@@ -44,6 +45,15 @@ module.exports = {
                 }
                 break;
             }
+            case 'printreport': {
```

Comment:
The ticket states:
"A flag/parameter to 'printReport' can be used when submitting a request that will return a report (such as when **submitting a synchronous job** or requesting a result from a previous job)"

Not sure, but getting the feeling we miss the combination where we supply this flag together with the flag for a synchronous job.

Reply:
I get the same feeling now. You mean it should be possible to print the report when the job is completed if it is synchronous, right?

Reply:
yes

Reply:
Now it is possible to print report when the job is completed in case of synchronous jobs